### DISCUSSION TITLE
Custom tool, field, etc .gitignore

### DISCUSSION THREAD

**Discussion by keizah7** [laravel/nova-issues#3589] [Q&A] [answered]
- Laravel Version: 8
- Nova Version:  v3.29.0 
- PHP Version: 7.4

### Description:
Creating custom tool or field using command doesn't have default .gitignore file like before
https://nova.laravel.com/docs/3.0/customization/tools.html#defining-tools

**Comment by crynobone** [comment, marked as answer]
This has been fixed in Laravel Nova 3.30.0 https://nova.laravel.com/releases/3.30.0
